1-Split-Array/Array split.py

```diff
@@ -0,0 +1,22 @@
+class num:
+	def __init__(self,positive,negative):
+		self.positive = positive
+		self.negative = negative
+def split(numbers):
+	for number in numbers:
+		number = int(number)
+		if number < 0:
+			negative.append(number)
+		else:
+			positive.append(number)
+	return num(positive,negative)
+positive = []
+negative = []
+numbers = (input("Enter numbers:")).split(",")
+numbers = split(numbers)
+if positive != []:
```

Comment:
why do you check arrays on emptiness? does it break if they will be empty?
if not, please remove redundant checking. 
instead of that add exception handling for all possible exceptions (use an experience from the Quick Sort lab)